concurrent/actor/eventloop/GlobalSystem.java

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
`GlobalSystem` effectively only has a timer / clock. Additionally, the clock is only used for handling cancellables (scheduled jobs) by `EventLoop`. Therefore, this class has been renamed to `Clock` and moved into `EventLoop`

Reply:
does it do "wall-clock" time or "logical-clock" time @lolski ?

Reply:
By default, wall-clock, but can be switched to logical clock @flyingsilverfin